cmd/state-installer/installer.go

```diff
@@ -32,7 +32,9 @@ func main() {
 		if panics.HandlePanics() {
 			exitCode = 1
 		}
-		events.WaitForEvents(1*time.Second, rollbar.Close)
+		if err := events.WaitForEvents(1*time.Second, rollbar.Close); err != nil {
+			logging.Warning("Failed to wait for rollbar to close: %v", err)
```

Comment:
I added a warning message to the logs for cases where we run into these timeouts.

I couldn't use `logging.Error`, because that will inject random `Rollbar: unknown token` printouts, as rollbar likely is already in a half-closed state...